serverless.yml

```diff
@@ -178,8 +178,20 @@ stepFunctions:
                       Next: defaultMapFail
                   Next: loadToObservationDb
                 loadToObservationDb:
-                  Type: Pass
+                  Type: Task
+                  Resource: arn:aws:lambda:${self:provider.region}:${self:custom.accountNumber}:function:aqts-capture-ts-loader-${self:provider.stage}-loadTimeSeries
+                  ResultPath: $.timeSeries
+                  TimeoutSeconds: 60
                   End: true
+                  Retry:
+                    - ErrorEquals:
+                      - States.TaskFailed
+                      IntervalSeconds: 35
+                      MaxAttempts: 2
+                      BackoffRate: 2
+                  Catch:
+                    - ErrorEquals:
+                      - States.ALL
```

Comment:
does this need 
                      Next: defaultMapFail
                  Next: defaultMapSuccess
to cover the Catch's next step and the Task's next step?

Reply:
@ayan-usgs maybe?

Reply:
I would suggest `Next: defaultMapFail` for the catch.

Reply:
added